app/main.py

```diff
@@ -2,0 +1,43 @@
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    @classmethod
+    def cleanup_dead_animals(cls) -> None:
+        cls.alive = [animal for animal in cls.alive if animal.health > 0]
+
+    def hide(self) -> None:
+        self.hidden = not self.hidden
+
+    def __str__(self) -> str:
+        return f"{{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}}"
+
+    def __repr__(self) -> str:
+        return str(self)
```

Comment:
The `__repr__` method currently returns the same string as `__str__`. Consider modifying it to ensure that the `Animal.alive` list is printed as a list of dictionaries.